frontend/.env.development

```diff
@@ -1,6 +1,6 @@
 REACT_APP_TITLE=SimpleReport
 REACT_APP_BACKEND_URL=http://localhost:8080
-REACT_APP_BASE_URL=http://localhost:3000
+REACT_APP_BASE_URL=http://localhost:3000/
```

Comment:
I think this might break the patient experience https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/utils/url.ts#L13. It will change the return type from `http://localhost:3000/app` to `http://localhost:3000//app`

Reply:
Removed here, though I do think this will make the state missing resources link break for local development. But that's not a huge issue, as it's not exactly on a critical path.